packages/addons/addon-depends/docker/cli/package.mk

```diff
@@ -0,0 +1,48 @@
+# SPDX-License-Identifier: GPL-2.0-or-later
+# Copyright (C) 2022 Team LibreELEC (https://libreelec.tv)
```

Comment:
```suggestion
# SPDX-License-Identifier: GPL-2.0
# Copyright (C) 2022-present Team LibreELEC (https://libreelec.tv)
```

Reply:
Using present isn't valid

Reply:
We decided long ago to use xxx-present for our complete codebase.
It avoids stupid and useless mass bumps of dates at 2023.01.01 or at first changes of the files in a particular year. 

Kodi for example uses a different approach, it just doesn't touch the dates anymore (or completely randomly) and have basically a 100% success rate at wrong dates at every file for example a complete [random file](https://github.com/xbmc/xbmc/blob/master/xbmc/CueDocument.cpp#L2) that should have at least 2005-2021 instead of the plain wrong 2005-2018. So basically every file contains wrong informations.

If anyone overtakes/forks the maintainment that is outside of LE he has to replace xxx-present with the year he overtook the maintainment. At that point our maintainment stops and we just keep the copyright of the changes at that timeframe we had maintained it. 
Easy for anyone to mass change it and we have a always correct date with zero effort.

Ofc this is not a homebrew idea, multi billion dollar companies using this approach for their OSS sources, so we can be sure some lawyers were involved in the decision making.